tests/unit/api/objectPutTagging.js

```diff
@@ -38,7 +38,7 @@ const testPutObjectRequest = new DummyRequest({
 function _checkError(err, code, errorName) {
     assert(err, 'Expected error but found none');
     assert.strictEqual(err.code, code);
```

Comment:
should be type as well, right?
```suggestion
    assert.strictEqual(err.type, code);
```
![image](https://user-images.githubusercontent.com/91462779/166888600-17d13d6b-178b-432a-95eb-a0d7b864908e.png)


You also need to update all `_checkError` functions in multiple places (you can use the `assert.strictEqual(err.code, code);` filter):
![image](https://user-images.githubusercontent.com/91462779/166888675-6ec20868-750e-4844-92ad-928e305de580.png)